test/common/common/BUILD

```diff
@@ -55,6 +55,15 @@ envoy_cc_fuzz_test(
     deps = ["//source/common/common:utility_lib"],
 )
 
+envoy_cc_fuzz_test(
+    name = "hash_fuzz_test",
+    srcs = ["hash_fuzz_test.cc"],
+    corpus = "hash_corpus",
+    # Fuzzer is stable, no bugs, simple test target; avoid emitting CO2.
```

Comment:
Did you consider letting it run for some amount of time on ClusterFuzz? What about catching regressions as we bump our version dependency?

Reply:
Could definitely do, I thought about following this PR with some regressions from local bazel/libfuzzer runs since it's relatively low risk, but I'll remove that tag and let it run!